main.py

```diff
@@ -0,0 +1,23 @@
+
+p1_scr = 0
+p2_scr = 0
+deck = {'1', '2', '3', '4', '5', '6', '7', '8', '9', '10'}
```

Comment:
why is the difference between {'1', '2'..} and ['1','2'..] ?

Reply:
Think on an anther way to create a list with 10 numbers. 
Let's assume you have 100000 numbers. what would you do then?

Reply:
{'1', '2'...} is a Set, which means it's unordered and unindexed. 
I put the numbers in a Set in order to be able to pop a random item using pop() method.
(It was more convenient way for me to do that for this matter, instead of using a List which is ordered and indexed).